src/meshapi/admin/inlines.py

```diff
@@ -135,3 +136,20 @@ class InstallInline(BetterInline):
     model = Install
     fields = ["status", "node", "member", "building", "unit"]
     readonly_fields = fields
+
+
+class InstallInlineNodePage(InstallInline):
+    add_button = True
+    reverse_relation = "primary_node"
+    bulk_add_button = True
+
+
+class InstallInlineBuildingPage(InstallInline):
+    add_button = True
+    reverse_relation = "building"
+    bulk_add_button = True
+
+
+class InstallInlineMemberPage(InstallInline):
+    add_button = True
+    reverse_relation = "member"
```

Comment:
Do we really need to make a separate class for each of these? Can't we set the `reverse_relation` when we declare/construct it later?

Reply:
I don't think so. You can see how we use it below, we just pass a class reference in a list attribute on the `ModelAdmin` class. I can't think of a way to change this property that doesn't pollute the `InstallInline` model (other than the one I've implemented here)

Reply:
Not even if we add a constructor?

Reply:
Oh no you're totally right. We're passed the model class in literally the first constructor param. Fixed